ocs_ci/node_scripts/pg_workaround.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+import subprocess
+import time
+
+all_pools = subprocess.check_output('ceph osd pool ls', shell=True)
+pools = all_pools.split('\n')
+print("Enabling pg_autoscaler")
+subprocess.check_output('ceph mgr module enable pg_autoscaler', shell=True)
```

Comment:
the pg autoscaler is now enabled by default by operator:
```
sh-4.2# ceph mgr module ls
{
    "enabled_modules": [
        "iostat",
        "pg_autoscaler",
        "prometheus",
        "restful",
        "rook"
    ],
```

Just connected to the env deployed here: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/809/console failed to apply old WA but after connected to that cluster I see it's enabled.